reports/pharmacy_first_report.Rmd

```diff
@@ -767,5 +768,116 @@ plot_measures(
   y_label = "Number of codes for PF conditions"
 ) + scale_color_manual(values = ethnicity_palette)
 ```
+```{r, message=FALSE, warning=FALSE, echo = FALSE}
+df_bsa_validation <- read_csv(here("lib", "validation", "data", "pf_consultation_validation_data.csv"))
+# Multiply by 0.4 to get 40% of data
+df_bsa_validation$count = df_bsa_validation$count * 0.4
```

Comment:
It might be useful to keep the original values in the dataframe, we could create a new column with the 40% `count_40pct`?

What do you think about also doing the rounding up here?

```suggestion
# Multiply by 0.4 to get 40% of data
df_bsa_validation %>%
  mutate(count_40pct = round(as.numeric(count * .4), digits = 0))
```

Reply:
Creating a new column might possibly complicate things, how about I create a separate dataframe with both count and count_40pct?